pkgs/servers/dict/gcide/default.nix

```
@@ -0,0 +1,105 @@
+{
+  autoreconfHook,
+  bison,
+  dict,
+  fetchurl,
+  flex,
+  lib,
+  libmaa,
+  stdenv,
+}:
+
+let
+  sed-scripts = {
+    check = ./sed-scripts/check.sed;
+    fixes = ./sed-scripts/fixes.sed;
+    post_webfilter = ./sed-scripts/post_webfilter.sed;
+  };
+
+  _major_debver = "0.48";
+  _debver = "${_major_debver}.5+nmu4";
+in
+stdenv.mkDerivation rec {
+  version = "0.54";
+
+  pname = "dict-gcide";
+  name = pname;
+  dbName = pname;
+
+  sourceRoot = "./${pname}";
+  srcs = [
+    (fetchurl {
+      url = "https://deb.debian.org/debian/pool/main/d/${pname}/${pname}_${_debver}.tar.xz";
```

```suggestion
      url = "https://deb.debian.org/debian/pool/main/d/dict-gcide/dict-gcide_${_debver}.tar.xz";
```
Otherwise overriding `pname` will result to invalid url.

---

Oof good catch!...  I'll have that fixed too _soon™_

> Side note; as I don't wanna clobber the diff you're currently reviewing I'll be awaiting a clear signal from ya before I force-push/squash changes